Build broken at master@dbdad86: gogs.go:167: undefined: gogs.ParseHook

See title; `make deps` isn't working from current head of `master`, dbdad86:
```
henrik@ip-10-0-42-21 ~/src $ git clone https://github.com/drone/drone.git                                                                                                                                                                
henrik@ip-10-0-42-21 ~/src $ cd drone
henrik@ip-10-0-42-21 ~/src/drone $ git log --oneline | head -n 1
dbdad86 fix SSL error when wget docker
henrik@ip-10-0-42-21 ~/src/drone $ docker build .
Sending build context to Docker daemon 18.93 MB
Sending build context to Docker daemon 
Step 0 : FROM google/golang
# Executing 1 build triggers
Trigger 0, RUN echo "This image is now deprecated. If you are not using Managed VMs, you should use the official Go Docker image located at: https://hub.docker.com/_/golang/"
[..]
Step 6 : RUN make deps build embed install
 ---> Running in c12dc240b66a
go get github.com/GeertJohan/go.rice/rice
go get -t -v ./...
code.google.com/p/goauth2 (download)
github.com/Bugagazavr/go-gitlab-client (download)
[...]
github.com/drone/drone/plugin/remote/gogs
# github.com/drone/drone/plugin/remote/gogs
plugin/remote/gogs/gogs.go:167: undefined: gogs.ParseHook
[...]
make: *** [deps] Error 2
The command '/bin/sh -c make deps build embed install' returned a non-zero code: 2

```

--- a/plugin/remote/gogs/gogs.go
+++ b/plugin/remote/gogs/gogs.go
@@ -164,7 +164,7 @@ func (r *Gogs) Deactivate(user *model.User, repo *model.Repo, link string) error
 func (r *Gogs) ParseHook(req *http.Request) (*model.Hook, error) {
 	defer req.Body.Close()
 	var payloadbytes, _ = ioutil.ReadAll(req.Body)
-	var payload, err = gogs.ParseHook(payloadbytes)
+	var payload, err = gogs.ParsePushHook(payloadbytes)
 	if err != nil {
 		return nil, err
 	}
@@ -181,7 +181,7 @@ func (r *Gogs) ParseHook(req *http.Request) (*model.Hook, error) {
 	return &model.Hook{
 		Owner:     payload.Repo.Owner.UserName,
 		Repo:      payload.Repo.Name,
-		Sha:       payload.Commits[0].Id,
+		Sha:       payload.Commits[0].ID,
 		Branch:    payload.Branch(),
 		Author:    payload.Commits[0].Author.UserName,
 		Timestamp: time.Now().UTC().String(),

